src/photos.cs

```diff
@@ -1005,9 +1005,15 @@ interface PHPhotoLibrary {
 
 	[iOS (9,1)]
 	[TV (10,0)]
+#if MONOMAC
+	[DisableDefaultCtor] // NS_UNAVAILABLE
```

Comment:
Hmm. They didn't fail locally, but I could have stale bits. Looks easy to fix. I'll get them here shortly.